.eslintrc

```diff
@@ -212,7 +212,10 @@ rules:
   no-eval: 2
 
   # Disallows adding to native types.
-  no-extend-native: 2
+  no-extend-native:
```

Comment:
I don't think we should be adjusting the global ESLint settings for the quirks of polyfills. 
How about creating a `/src/static/js/modules/polyfill/` path where the `array-foreach-polyfill.js` module would live and put a `.eslintrc` file in `/src/static/js/modules/polyfill/.eslintrc`with the following contents:

```
# ESLint overrides for polyfills.
no-extend-native:
  - 2
  - exceptions:
    - Array
```